src/rapidfuzz/fuzz_py.py

```diff
@@ -335,7 +336,7 @@ def partial_ratio_alignment(
     if res.score < score_cutoff:
         return None
 
-    if len(s1) <= len(s2):
+    if len1 <= len2:
```

Comment:
This change is fine. It shouldn't make a relevant difference though and is part of the fallback implementation